src/components/Materias.js

```diff
@@ -82,6 +82,10 @@ const Materias = ({ materias, materiaSelected, setCodigoSelected, partialLoading
                   borderRadius={6}
                   p={8}
                   bg="white"
+                  _hover={{
+                    cursor: "pointer",
+                    bg: "whiteAlpha.700",
+                  }}
```

Comment:
Se le puede agregar una transicion para que sea mas ameno a los ojos, me parece

```suggestion
                  }}
                transitionDuration="100ms"
```